### ISSUE TITLE
Hatchery won't auto before beating misty.

### ISSUE THREAD

**Issue by Xross98** [Ephenia/Pokeclicker-Scripts#124]
Trying to do a challenge run where I get all pokemon shiny before beating the next gym, I need the hatchery to auto the eggs otherwise this will take a dang while, I'm assuming that the hatchery auto-ing is tied to having level 100's available? maybe?

Any way around this would be grand thx.

**Comment by Ephenia** [comment]
> I'm assuming that the hatchery auto-ing is tied to having level 100's available? maybe?
Yes, since the Auto Hatchery is just automating what a human could do normally, albeit not necessarily realistically. Pokémon normally need to be level 100 to be able to be placed into the Hatchery and the script follows that.

I'm sure someone could offer some solutions via doing a bit of source modification and even through the Auto Hatchery script itself too.
